test/new-e2e/utils/e2e/client/agent.go

```diff
@@ -98,26 +102,33 @@ func (a *Agent) IsReady() (bool, error) {
 	return a.Status().isReady()
 }
 
+func (a *Agent) IsHealthy() (bool, error) {
```

Comment:
I don't think this method should belong to this file but only when testing the health. I would like to avoid seeing this function calls everywhere in test "just in case". What do you think?